packages/react-resize-handle/src/hooks/useResizeHandle.ts

```diff
@@ -64,19 +66,25 @@ export type UseResizeHandleParams = {
    * A function that will be called to get the value that will be set as the aria-valuetext attribute on the resize handle.
    * Use this for localization.
    */
-  getA11ValueText?: (value: number) => string | undefined;
+  getA11ValueText?: (value: number, unit: string) => string | undefined;
```

Comment:
Curious to know if narration for aria-valuetext works with viewport units

Reply:
<img width="590" alt="image" src="https://github.com/user-attachments/assets/55c33de9-10c4-42e9-8197-75e6a191a0d3" />

Works the same way as for pixels:

<img width="656" alt="image" src="https://github.com/user-attachments/assets/b934ec99-1e00-48d1-a739-4117e307c7f0" />

<img width="655" alt="image" src="https://github.com/user-attachments/assets/d1ce1629-eed0-463c-af03-0ef5716cb4c6" />

But I would not call VoiceOver experience great